### DISCUSSION TITLE
Maven repository

### DISCUSSION THREAD

**Discussion by Plyha** [dynomake/libretranslate-java#5] [Ideas]
Hi, are you planning to publish your library in https://mvnrepository.com/ ?
I think it would be great if this library was there.

**Comment by dynomake** [comment]
Future releases will be there instead of Clojars. Thanks for the idea.

**Comment by Plyha** [comment]
When is the release planned with ENUM Ukrainian fix and maven repository?  Thanks for your hard work !

And pay attention to this)
https://github.com/suuft/libretranslate-java/discussions/1#discussioncomment-4470011
